### PR TITLE
Nodejs week2

### PR DESCRIPTION
**Pull request by Dima-Kutaini**

### CODE REVIEW

File: src/backend/api/meals.js [hunk 1]
```diff
@@ -1,15 +1,90 @@
...
 module.exports = router;
+
+//Returns all meals
+mealsRouter.get('/', async (req, res) => {
+  try {
+    const meals = await knex.select('*').from('meals');
+    res.json(meals);
+  } catch (error) {
+    console.log(error);
+    res.status(500).json({error:'error occurred!'});
```
Comment (babak-f) [inline src/backend/api/meals.js:26]: The status code 500 already implies that an internal server error has occurred. So no need to add the 'error occurred' part.

File: src/reservation.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+//   }
+//});
+
+//Returns all reservations
+reservationRouter.get('/',async(req,res)=>{
+    try{
+        const allReservations= await knex.select('*').from('Reservation'); 
+        res.json(allReservations); 
+
+    }catch (error) {
```
Comment (babak-f) [inline src/reservation.js:23]: Remember to format your code before committing. It's a very good habit to get used to when working in teams. You could also set VS code to format automatically on save.

File: src/reservation.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    console.log(error);
+    res.status(500).send('Error!');
+  }
+});
+
+//Deletes the  reservation by id
+reservationRouter.delete('/:id', async (req, res) => {
+  const id = parseInt(req.params.id);
+  try {
+    const deletedReservation = await knex('Reservation').where({ id: id }).del();
```
Comment (babak-f) [inline src/reservation.js:78]: You don't need the `: id` part when the two sides have the same name ('id').

**Review by babak-f — APPROVED** [review]
Very good job. Left a few minor comments.

**Review by babak-f — APPROVED** [review]
Very good job. Left a few minor comments.

File: src/reservation.js (hunk 3 above)
Comment (Dima-Kutaini) [reply src/reservation.js:78]: thanks for the review i will update  my homework according to your commits :)

**Pull request merged**